tests/integration/targets/lookup_hashi_vault/lookup_hashi_vault/tasks/tinyproxy_server.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Install tinyproxy
+  shell: |
+    apt-get update
+    DEBIAN_FRONTEND=noninteractive apt-get --assume-yes install tinyproxy
+  changed_when: false
```

Comment:
This isn't going to be portable. I recommend looking at [the `package` module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_module.html), as this should pick the right package provider for the given OS.

The default testing containers use Ubuntu, and we use those in the CI, so it MUST work in Ubuntu, but where possible, when someone runs `ansible-test` locally, either against their local env or with a different container, we should strive for it to work.

In the case where we try to make it portable, but know for certain of environments where it can't, let's say we know it doesn't work on MacOS for example, then we need to gate the entire class of tests so that on those platforms, a test run can still succeed, even if skips some tests that it can't execute (there are some conditionals like that already within the tests for some RedHat versions, etc.). If you tried that and hit errors link me to a test run.

There is also an issue with using the `package` or `apt` modules on Ubuntu/Debian, when running in a version of python that doesn't match the "system" version of python (which _will happen_ in CI). Check `vault_download.yml` to see how I got around that.

We'll also want to add and then notify a handler that will uninstall this package at the end of the tests (not always running in a container).

Reply:
done

In fact on my first try I used "package" but as you describe it failed. I didn't saw you already have a workaround.